test/controllers/ConfirmTransferControllerSpec.scala

```diff
@@ -471,28 +478,30 @@ class ConfirmTransferControllerSpec extends FrontEndTestHelper {
       wiremockServer.getAllServeEvents.size() should equal(4)
     }
 
-    "render the confirm transfer 'already confirmed' page with an authenticated standard user if export status is 'Completed'" in {
-      val controller = instantiateConfirmTransferController(getAuthorisedSecurityComponents)
-      setConsignmentStatusResponse(app.configuration, wiremockServer, exportStatus = Some("Completed"))
-      setConsignmentTypeResponse(wiremockServer,"standard")
-
-      val transferControllerPage = controller.confirmTransfer(consignmentId)
-        .apply(FakeRequest(GET, f"/consignment/$consignmentId/confirm-transfer").withCSRFToken)
-      val confirmTransferPageAsString = contentAsString(transferControllerPage)
-
-      playStatus(transferControllerPage) mustBe OK
-      contentType(transferControllerPage) mustBe Some("text/html")
-      headers(transferControllerPage) mustBe TreeMap("Cache-Control" -> "no-store, must-revalidate")
-      confirmTransferPageAsString must include(
-        s"""href="/consignment/$consignmentId/transfer-complete">Continue""".stripMargin)
-      confirmTransferPageAsString must include("Your transfer has already been completed")
+    forAll(consignmentStatuses) { consignmentStatus =>
+      s"render the confirm transfer 'already confirmed' page with an authenticated standard user if export status is '$consignmentStatus'" in {
+        val controller = instantiateConfirmTransferController(getAuthorisedSecurityComponents)
+        setConsignmentStatusResponse(app.configuration, wiremockServer, exportStatus = Some(consignmentStatus))
+        setConsignmentTypeResponse(wiremockServer, "standard")
+
+        val transferControllerPage = controller.confirmTransfer(consignmentId)
+          .apply(FakeRequest(GET, f"/consignment/$consignmentId/confirm-transfer").withCSRFToken)
+        val confirmTransferPageAsString = contentAsString(transferControllerPage)
+
+        playStatus(transferControllerPage) mustBe OK
+        contentType(transferControllerPage) mustBe Some("text/html")
+        headers(transferControllerPage) mustBe TreeMap("Cache-Control" -> "no-store, must-revalidate")
+        confirmTransferPageAsString must include(
+          s"""href="/consignment/$consignmentId/transfer-complete">Continue""".stripMargin)
+        confirmTransferPageAsString must include("Your transfer has already been completed")
+      }
     }
 
     "render the confirm transfer 'already confirmed' page with an authenticated user if the standard user navigates back to the " +
```

Comment:
Should not this test be updated to cover all the statuses: `Failed`, `InProgress`, `Completed` as with the other tests